data/json/dokubib.json

```diff
@@ -25,14 +25,23 @@
       {
         "en": "Historic photograph"
       },
+      {
+        "de": "Historisches Foto"
+      },
       {
         "en": "Local history"
       },
       {
-        "en": "St. Moritz"
```

Comment:
@danielasubotic You have kept the`de` value for `St. Moritz` (line#41). Is removing the `de` version intentional?